packages/libllvm/build.sh

```diff
@@ -4,48 +4,38 @@ TERMUX_PKG_LICENSE="Apache-2.0, NCSA"
 TERMUX_PKG_LICENSE_FILE="llvm/LICENSE.TXT"
 TERMUX_PKG_MAINTAINER="@finagolfin"
 # Keep flang version and revision in sync when updating (enforced by check in termux_step_pre_configure).
-LLVM_MAJOR_VERSION=21
-TERMUX_PKG_VERSION=${LLVM_MAJOR_VERSION}.1.6
+TERMUX_PKG_VERSION=21.1.6
+TERMUX_PKG_REVISION=1
 TERMUX_PKG_SHA256=ae67086eb04bed7ca11ab880349b5f1ab6f50e1b88cda376eaf8a845b935762b
 TERMUX_PKG_AUTO_UPDATE=false
-TERMUX_PKG_SRCURL=https://github.com/llvm/llvm-project/releases/download/llvmorg-$TERMUX_PKG_VERSION/llvm-project-${TERMUX_PKG_VERSION}.src.tar.xz
+TERMUX_PKG_SRCURL=https://github.com/llvm/llvm-project/releases/download/llvmorg-${TERMUX_PKG_VERSION}/llvm-project-${TERMUX_PKG_VERSION}.src.tar.xz
 TERMUX_PKG_HOSTBUILD=true
 TERMUX_PKG_RM_AFTER_INSTALL="
 bin/ld64.lld.darwin*
 lib/libgomp.a
 lib/libiomp5.a
 share/man/man1/lit.1
 "
-TERMUX_PKG_DEPENDS="libc++, libffi, libxml2, ncurses, zlib, zstd"
-TERMUX_PKG_BUILD_DEPENDS="binutils-libs"
+TERMUX_PKG_DEPENDS="libc++, libffi, libxml2, ncurses, zlib"
```

Comment:
You sure about removing the `zstd` dependency? I see `libLLVM.so` linked against it in the package.

Reply:
The `zstd` dependency was introduced together with the `llvmgold` subpackage.
https://github.com/termux/termux-packages/commit/8552671238e002790e258747a09872f641bbc927
There was no further explanation given for it so I assumed it was only needed for that.
Let me go check the build documentation for LLVM.
I suspect you are correct though.